src/services/report_service.js

```diff
@@ -6,25 +6,38 @@ export default (easypostClient) =>
 
     /**
      * Create a report.
-     * @param {object} prams
-     * @returns {Pickup}
+     * @param {object} params
+     * @returns {Report}
      */
     static async create(params) {
+      const url = `${this._url}/${params.type}`;
+      return this._create(url, params);
+    }
+
+    /**
+     * Retrieve a list of all reports associated with the API key.
+     * @param {string} type
+     * @param {object} params
+     * @returns {Report[]}
+     */
+    static async all(type, params = {}) {
```

Comment:
This is an unintended breaking change. Let's just pull `type` out of `params` like it already was. All our other libs continue doing this.

Reply:
Looks like you added this during the rewrite last week? You're saying you want it removed?
<img width="654" alt="image" src="https://user-images.githubusercontent.com/17054780/219460400-b74b0c4d-1e48-43d9-b208-35694d0e07e0.png">

Reply:
Huh. If I added it, it was a mistake. Can you verify I actually did this or was it like this before?

TLDR: Yes, we should have this just pull out of `params` since you can also pass in all the other bits like IDs and timestamps

Reply:
Current version does it this way: https://github.com/EasyPost/easypost-node/blob/6d013bb1948b2162a14528a1d354b7ddce461631/src/resources/report.js#L51

Reply:
Dang... I'd suggest to put it in params and call it out in the CHANGELOG. All our other libs operate this way, it's much more extensible, and it would be super weird to have a single function that doesn't follow the pattern the rest of the lib does.

We also talked about this internally and I believe the consensus was to put it in `params`.

Reply:
Tale as old as time, C# does it one way, Java/PHP the other:

https://github.com/EasyPost/easypost-csharp/blob/aac9873379f710f8774e59f1d70f1141adeb626b/EasyPost/Services/ReportService.cs#L55

https://github.com/EasyPost/easypost-java/blob/351090eb9b3d2b836f36b683bed6bcc5d46edce4/src/main/java/com/easypost/service/ReportService.java#L85

https://github.com/EasyPost/easypost-php/blob/4818cc05c1fba77052cfebff7fa68143f7397412/lib/EasyPost/Service/ReportService.php#L38

Reply:
(Yes, I did sing that.)

Reply:
Anyway, thoughts? Introduce a breaking change, or keep it the way it is, since apparently cross-library consistency is out the window currently.

Reply:
I believe I answered your question above

Reply:
Lul, does this happen on Java/PHP too?

Reply:
OH, delete `type` from params after crafting the URL